### DISCUSSION TITLE
Does this require any setup?

### DISCUSSION THREAD

**Discussion by sriram-rao** [tadashi-aikawa/obsidian-various-complements-plugin#326] [Q&A]
Hello, 
I downloaded the plugin just today and I haven't been able to use it in Obsidian at all, not even with the command to manually trigger suggestions. I've tried reinstalling the plugin, restarting Obsidian, tried changing settings that even had a remote chance to interrupt its startup. Am I missing something really obvious or there some setup that I may have missed? 

Really looking forward to using the plugin, it looks super-useful.

\- Sriram

**Comment by tadashi-aikawa** [comment]
Hi, @sriram-rao :)

Does that reproduce in the sandbox vault? If so, could you provide the steps to reproduce the issue in the sandbox vault and explain what you expected to happen?

**Comment by sriram-rao** [comment]
Hi @tadashi-aikawa! As I was typing out the steps here, I had the documentation page open on the side and I just got what it's supposed to do. It is working exactly as intended. 
Thanks for helping :)

**Reply by tadashi-aikawa** [reply]
I’m glad to hear that!
